contentmgr/replication.go

```diff
@@ -841,24 +809,42 @@ func (cm *ContentManager) addContentToStagingZone(ctx context.Context, content u
 	cm.BucketLk.Lock()
 	defer cm.BucketLk.Unlock()
 
-	blist, ok := cm.Buckets[content.UserID]
-	if !ok {
-		b, err := cm.newContentStagingZone(content.UserID, content.Location)
+	var zones []util.Content
+	if err := cm.DB.Find(&zones, "not active and pinning and aggregate and user_id = ?", content.UserID).Error; err != nil {
```

Comment:
cant we do something like this `not active and pinning and aggregate and user_id =? and size + ? <= ?", user_id, content.Size, max content`

Reply:
yep good catch! this could be simplified even further after the incremental size tracking